src/connection.js

```diff
@@ -929,6 +997,19 @@ class Connection extends EventEmitter {
     process.nextTick(boundTransitionTo, this.STATE.SENT_NTLM_RESPONSE);
   }
 
+  sendFedAuthResponsePacket() {
+    const accessTokenLen = Buffer.byteLength(this.fedAuthInfo.token.accessToken, 'ucs2');
+    const data = new WritableTrackingBuffer(4 + accessTokenLen);
+    data.writeUInt32LE(accessTokenLen + 4);
+    data.writeUInt32LE(accessTokenLen);
+    data.writeString(this.fedAuthInfo.token.accessToken, 'ucs2');
+    this.messageIo.sendMessage(TYPE.FEDAUTH_TOKEN, data.data);
+
+    const boundTransitionTo = this.transitionTo.bind(this);
+    // sent the fedAuth token message, the rest is similar to standard login 7
+    process.nextTick(boundTransitionTo, this.STATE.SENT_LOGIN7_WITH_STANDARD_LOGIN);
```

Comment:
What do you think about using an arrow function here, instead of using `Function.prototype.bind`?

```js
process.nextTick(() => {
  this.transitionTo(this.STATE.SENT_LOGIN7_WITH_STANDARD_LOGIN);
});
```